api/helpers.go

```diff
@@ -62,30 +62,54 @@ func findEntry(ctx context.Context, root plugin.Entry, segments []string) (plugi
 	return curEntry, nil
 }
 
-func getPathFromRequest(r *http.Request) (string, *errorResponse) {
+func getFullPathFromRequest(r *http.Request) (string, *errorResponse) {
```

Comment:
Hmm, what was wrong with `getPathFromRequest` ?

Reply:
getWashPathFromPath seemed weird, and getWashPathFromFullPath + getFullPathFromRequest has symmetry.